clms-angular/src/app/modules/admin/question-bank/admin-question-bank.module.ts

```diff
@@ -0,0 +1,51 @@
+import {NgModule} from "@angular/core";
```

Comment:
created this admin question bank module so we only deal with question bank related stuff here